script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -801,14 +808,15 @@ My whole body shakes with a feeling far stronger than that.
 [sha:2044cd5ed28fb93c562208d484661cd8d9ef9964]{
 -- Page 706, Offset 10569.
 -- ヴローヴの手には鉈が握られていた。必殺の巨槍を放った後でさえ、この吸血鬼は、俺が“来る”と確信し、さらなる迎撃を選択したのか。
-A machete is grasped in Vlov's hand.
- Even after unleashing his giant, deadly lance,
- this vampire was convinced that I was coming and chose to intercept me.
+Vlov holds a machete in his hand.
+ Even after unleashing his colossal lance,
+ this vampire was fully convinced that I was still coming,
+ and chose to have one final counterattack lie in wait for me.
```

Comment:
Don't love lie in wait here. Just waiting for me, or "still coming, and chose to keep one final counterattack up his sleeve"